internal/changestream/testing/modelsuite.go

```diff
@@ -0,0 +1,44 @@
+// Copyright 2023 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package testing
+
+import (
+	gc "gopkg.in/check.v1"
+
+	"github.com/juju/juju/core/changestream"
+	coredatabase "github.com/juju/juju/core/database"
+	"github.com/juju/juju/domain/schema/testing"
+)
+
+// ModelSuite is used to provide a sql.DB reference to tests.
+// It is pre-populated with the model schema.
+type ModelSuite struct {
```

Comment:
Why not make a base suite that goes in both of these so as to avoid the duplication?